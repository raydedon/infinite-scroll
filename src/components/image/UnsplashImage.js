import React from 'react';
import classes from './unsplash.scss';

const UnsplashImage = ({ url }) => (
    <div className={classes.container}>
        <img src={url} className={classes.responsiveImg}/>
    </div>
);

export {UnsplashImage};
export default UnsplashImage;
