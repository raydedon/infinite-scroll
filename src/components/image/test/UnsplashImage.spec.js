import React from 'react';
import { shallow } from 'enzyme';
import UnsplashImage from '../UnsplashImage';
import '../../../setupTests'
let wrapped = shallow(<UnsplashImage url='url'/>);
describe('UnsplashImage', () => {
    it('should render the UnsplashImage Component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });
});
