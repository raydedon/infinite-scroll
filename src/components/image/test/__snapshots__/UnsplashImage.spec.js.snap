// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnsplashImage should render the UnsplashImage Component correctly 1`] = `
<div
  className="container"
>
  <img
    className="responsiveImg"
    src="url"
  />
</div>
`;
